src/renderer/fonts/font_loader.rs

```diff
@@ -230,6 +230,7 @@ mod test {
     }
 
     #[test]
+    #[ignore]
```

Comment:
I agree with @j4qfrost I think this should either be left enabled or swapped to use a different font rather than deleting the test? What do you think

Reply:
I looked it up, Monospace is installed for all Linux systems.

Reply:
Cool, I'll change it to monospace

Reply:
Why were you checking for DejaVu Serif though? Shouldn't you have been checking for DejaVu Mono?